webapp/app/css/treeherder.css

```diff
@@ -972,6 +972,17 @@ ul.failure-summary-list li .btn-xs {
     margin-bottom: -1px;
 }
 
+.pinboard-related-bugs-btn > .pinboard-related-bugs-link,
+.pinboard-related-bugs-btn:hover > .pinboard-related-bugs-link:hover {
+    border-top-right-radius: 3px !important;
+    border-bottom-right-radius: 3px !important;
+}
+
+.pinboard-related-bugs-btn:hover > .pinboard-related-bugs-link {
+    border-top-right-radius: 0 !important;
+    border-bottom-right-radius: 0 !important;
+}
```

Comment:
I usually try to avoid `!important`, although I don't know off hand why I used it here. I think there were other css rules with superceeding css specificity and I didn't come up with a readable alternative.

The CSS here is to make the rounded edges look nice when hovering over the related bug input. Steps: enter and accept a related bug. Then hover over the bug number. Before, the right side had hard edges, now it will look nice and rounded. Then hover over the X button, it should now only be rounded on the right of the X button, as before.